src/components/modals/AddressModal.tsx

```diff
@@ -96,8 +98,16 @@ export class AddressModalComponent extends React.Component<Props, State> {
     }
   }
 
-  setStatusLabel = (status: string) => {
-    this.setState({ statusLabel: status })
+  setShowSpinner = (showSpinner: boolean) => {
+    this.setState({ showSpinner })
+  }
+
+  setValidLabel = (validLabel: string | undefined) => {
+    this.setState({ validLabel })
+  }
+
+  setErrorLabel = (errorLabel: string | undefined) => {
```

Comment:
Get rid of these wrappers. Just call this.setState directly. one-liner wrappers not too useful here.